lessons/module-3/unit-testing-react.md

```diff
@@ -313,20 +298,40 @@ const Grocery = ({ name, quantity, notes, purchased, starred, onPurchase, onStar
 export default Grocery;
 ```
 
-Running the tests now, you should see a failure for the snapshot test, with the
-differences between the two snapshots in the console. Here is where you have to
-consider, 'am I ok with those changes?' If so, you can update the tests by
-typing `u` while `Jest` is still in watch mode. Alternatively, you can restart
-your test suite and run `npm test --updateSnapshot`.
+Run:
+
+```npm test```
```

Comment:
Should only need single ticks.

Reply:
You may be right.  I am still learning the specific standards and best use of the ticks and hashes.  I have been considering that if the lesson is meant for students to code along, then when terminal commands are expected, the triple back-ticks as a wrapper give the gray back-ground and help the students by being visual place-holders as they work along:  "I've worked to this command... now I've worked to this command."